internal/capability/caldera/caldera.go

```diff
@@ -43,5 +54,100 @@ func (capability *calderaCapability) Execute(
 	variables cacao.Variables) (cacao.Variables, error) {
 
 	log.Info("Successfully called execute on the caldera capability")
```

Comment:
Do the other capabilities have logging like this? Might be nice to remove this to avoid excessive logging